specs/windows/shellbags.table

```diff
@@ -0,0 +1,16 @@
+table_name("shellbags")
+description("Shows directories accessed via Windows Explorer.")
+schema([
+    Column("sid", TEXT, "User SID"),
+    Column("source", TEXT, "Shellbags source Registry file"),
+    Column("path", TEXT, "Directory name."),
+    Column("modified_time", INTEGER, "Directory Modified time."),
+    Column("created_time", INTEGER, "Directory Created time."),
+    Column("accessed_time", INTEGER, "Directory Accessed time."),
+    Column("mft_entry", BIGINT, "Directory mft entry."),
+    Column("mft_sequence", INTEGER, "Directory mft sequence."),
```

Comment:
Perhaps more clear for those less familiar
```suggestion
    Column("mft_entry", BIGINT, "Directory master file table entry."),
    Column("mft_sequence", INTEGER, "Directory master file table sequence."),
```

Reply:
changed to `master file table`